components/gitpod-cli/cmd/credential-helper.go

```diff
@@ -145,39 +146,70 @@ func parseHostFromStdin() string {
 	return host
 }
 
-func logDebug(v ...interface{}) {
-	if os.Getenv("CREDENTIAL_HELPER_DEBUG_LOG") == "true" {
-		log.Println(v...)
+func parseProcessTree() (repoUrl string, gitCommand string) {
```

Comment:
Any chance we could write some unit tests for this? Not just for the actual test but that would also help to understand the function because we would have a list of some example inputs and expected outputs.

Sure, the problem here is to mock the `readProc` function for the test, right? Any ideas?